Documentation/design-docs/standalone-gc-loading.md

```diff
@@ -0,0 +1,222 @@
+# Standalone GC Loader Design
+
+Author: Sean Gillespie (@swgillespie) - 2017
+
+A standalone GC (or "local GC", as it is often called) initiaive has
```

Comment:
This may rather be in PR description. Not in the design document that should stay relevant over time.

Reply:
ok!